org.eclipse.lsp4xml/src/test/resources/catalogs/catalog-liferay.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
```

Comment:
It's not a XML catalog. Please update this file with 

```xml
<?xml version="1.0"?>
<catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
	<system
		systemId="http://www.liferay.com/dtd/liferay-service-builder_7_2_0.dtd"
		uri="../dtd/liferay-service-builder_7_2_0.dtd" />
</catalog>
```
To check that the XML catalog is working and uses the local liferay-service-builder_7_2_0.dtd, disconnect your Internet and try to execute your test.

Reply:
Sorry, just got it. the pr has been updated.